lib/train/extras/os_common.rb

```diff
@@ -96,6 +98,8 @@ def detect_family
       # TODO: extend base implementation for detecting the family type
       # to Windows and others
       case uname_s
+      when /unrecognized command verb/
+        @platform[:family] = 'openvms'
```

Comment:
Since we cannot expect that every non-uname system is openvms I would prefer to set it to nil here?